php/oop/examples/animals.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+abstract class Animal
+{
+
+    public function makeSound()
+    {
+
+    }
+
+    public function move()
+    {
+    }
+}
+
+class wildAnimal extends Animal
+{
+
+}
+
+trait givingbirth
```

Comment:
It's better to start your classes/traits/interfaces with an uppercase characters